modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -32,42 +32,43 @@ uniform float project_uCoordinateSystem;
 uniform float project_uScale;
 uniform bool project_uWrapLongitude;
 uniform float project_uAntimeridian;
-uniform vec3 project_uPixelsPerMeter;
-uniform vec3 project_uPixelsPerDegree;
-uniform vec3 project_uPixelsPerUnit;
-uniform vec3 project_uPixelsPerUnit2;
+uniform vec3 project_uDistancePerMeter;
+uniform vec3 project_uDistancePerDegree;
+uniform vec3 project_uDistancePerUnit;
+uniform vec3 project_uDistancePerUnit2;
```

Comment:
Side note: This `perUnit2` is really a terribly unhelpful name... Anything that can be done while we are changing things?